### DISCUSSION TITLE
Generate XML file based upon a defined XSD Schema

### DISCUSSION THREAD

**Discussion by librafrog** [anansi-project/comicinfo#59] [Q&A] [answered]
I have a rather dull ❓ regarding how exactly can we create a simple xml file based on the comicinfo schema (XSD). Should we declare it somehow at the begginning of the file? A rather simple example would do the trick, if any had tried it.

- I'm using Visual Studio Code + an extension **XML** to beautify the code.

**Comment by majora2007** [comment, marked as answer]
I'm not sure I understand what you are asking. The schema is defined. You can follow the schema and use any tool to generate based from the schema or you can use existing taggers like Manga Manager or Comic Tagger to generate an example.

**Reply by librafrog** [reply]
ok, I'll try that; thanks
